homeassistant/components/ezviz/camera.py

```diff
@@ -1,197 +1,253 @@
+        self._idx = idx
+        self._ffmpeg = hass.data[DATA_FFMPEG]
+        self._local_rtsp_port = local_rtsp_port
+        self._ffmpeg_arguments = ffmpeg_arguments
+
+        self._serial = self.coordinator.data[self._idx]["serial"]
+        self._name = self.coordinator.data[self._idx]["name"]
+        self._local_ip = self.coordinator.data[self._idx]["local_ip"]
 
     @property
     def device_state_attributes(self):
         """Return the Ezviz-specific camera state attributes."""
         return {
-            # if privacy == true, the device closed the lid or did a 180° tilt
-            "privacy": self._privacy,
+            # Camera firmware version
+            "sw_version": self.coordinator.data[self._idx]["version"],
+            # Camera firmware version update available?
+            "upgrade_available": self.coordinator.data[self._idx]["upgrade_available"],
+            # if privacy == true, the device closed the lid or did a 180� tilt
+            "privacy": self.coordinator.data[self._idx]["privacy"],
+            # if sleep == true, the device is sleeping?
+            "sleep": self.coordinator.data[self._idx]["sleep"],
             # is the camera listening ?
-            "audio": self._audio,
+            "audio": self.coordinator.data[self._idx]["audio"],
             # infrared led on ?
-            "ir_led": self._ir_led,
+            "ir_led": self.coordinator.data[self._idx]["ir_led"],
             # state led on  ?
-            "state_led": self._state_led,
+            "state_led": self.coordinator.data[self._idx]["state_led"],
             # if true, the camera will move automatically to follow movements
-            "follow_move": self._follow_move,
-            # if true, if some movement is detected, the app is notified
-            "alarm_notify": self._alarm_notify,
+            "follow_move": self.coordinator.data[self._idx]["follow_move"],
+            # if true, notification schedule(s) are configured
+            "alarm_schedules_enabled": self.coordinator.data[self._idx][
+                "alarm_schedules_enabled"
+            ],
             # if true, if some movement is detected, the camera makes some sound
-            "alarm_sound_mod": self._alarm_sound_mod,
+            "alarm_sound_mod": self.coordinator.data[self._idx]["alarm_sound_mod"],
             # are the camera's stored videos/images encrypted?
-            "encrypted": self._encrypted,
+            "encrypted": self.coordinator.data[self._idx]["encrypted"],
             # camera's local ip on local network
-            "local_ip": self._local_ip,
-            # from 1 to 9, the higher is the sensibility, the more it will detect small movements
-            "detection_sensibility": self._detection_sensibility,
+            "local_ip": self.coordinator.data[self._idx]["local_ip"],
+            # camera's battery level if battery camera
+            "battery_level": self.coordinator.data[self._idx]["battery_level"],
```

Comment:
Battery level and similar measurements that aren't controlled by a service should be a sensor entity.

Reply:
Done.